wave_vision/src/fast_detector.cpp

```diff
@@ -0,0 +1,87 @@
+// Libwave Headers
+#include "wave/vision/fast_detector.hpp"
+
+namespace wave {
+
+FASTDetector::FASTDetector(const std::string &config_path) {
+    // Extract parameters from .yaml file.
+    ConfigParser parser;
```

Comment:
No no, you're right. I didn't include that implementation. I will add two more constructors; a default one and one which takes in the configuration arguments directly.